qemu/tests/cfg/hugepage_reset.cfg

```diff
@@ -0,0 +1,15 @@
+- hugepage_reset:
+    type = hugepage_reset
+    only x86_64
+    start_vm = no
+    pre_command = 'echo 3 > /proc/sys/vm/drop_caches && echo 1 > /proc/sys/vm/compact_memory'
+    mem = 4096
+    origin_nr = 8
+    # Please set hugepage in kernel command line before this test:
+    # default_hugepagesz=1G hugepagesz=1G hugepages=8
+    expected_hugepage_size = 1048576
```

Comment:
could you help to update this to "default_hugepagesz=2G hugepagesz=2G hugepages=8" for ppc ?  and expecte_hugepage_size=2097152

Reply:
@MiriamDeng Updated, thanks.

Reply:
Sorry, @vivianQizhu, it should be like this "default_hugepagesz=1G hugepagesz=1G hugepages=8" so need to update it for ppc specially.

Reply:
For reference, 
https://polarion.engineering.redhat.com/polarion/#/project/RedHatEnterpriseLinux7/workitem?id=RHEL-152470

Reply:
By the way,I changed the cfg as followings by manual and tried it on ppc, it works as expected.Please help to update it and thanks.
"default_hugepagesz=1G hugepagesz=1G hugepages=8" 
 (1/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.hugepage_reset: PASS (188.76 s)
 (2/2) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.hugepage_reset.on_numa_node: PASS (180.88 s)